test/js/node/util/parse_args/utils.test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const tape = require('tape');
```

Comment:
Thank you for porting all these tests. Could we use `bun:test` instead of `tape`?

Reply:
Hi, that was my plan initially but I stalled out because it's a significant pain to manually re-write the tests due to different functionality and interfaces of the APIs. I'm not sure when I'll have time & motivation to complete the work, so either someone picks up this branch and helps to re-write the tests, or we merge the work without including the tests, or this will just become a stale PR and someone else with more time and motivation will re-do all the work to bring in parseArgs into bun.

Reply:
I'm interested in this feature, I can help porting the tests if you want

Reply:
I will spend some time analyzing your conversion and will merge it into my PR, thanks for taking this on!

Reply:
@otgerrogla I've opened a PR with your changes, but the change to the exported object bothers me, more info: https://github.com/gamedevsam/bun/pull/1/files#r1391944352

I'll spend some time to see if this can be avoided.

Reply:
Here's the PR that removes the internal functions from the exported module: https://github.com/otgerrogla/bun/pull/1/files

Once you merge it into your branch, I will merge it into my branch and we should be good to merge this into Bun!